### DISCUSSION TITLE
Github copilot wrongly charged twice

### DISCUSSION THREAD

**Discussion by MohamadTahir** [community/community#90057] [New to GitHub]
Hello, I was trying to purchase Copilot for 1 year but I ended up getting billed for ($10.00/month Nov 3, 2023 - Dec 2, 2023) which is last year's transaction?!

**Comment by NipulM** [comment]
Hello 👋🏼

Contact GitHub Support:
Visit the GitHub Support website [GitHub Support](https://support.github.com/) and look for options to contact them. You may find a "Contact Us" or "Submit a Request" link.
Provide details about the incorrect charge, including transaction dates, amounts, and any relevant account information.

**Reply by MohamadTahir** [reply]
I submitted a ticket 3 days ago and I am still waiting for a reply.

**Reply by MohamadTahir** [reply]
it has been over a week now and I still haven't gotten a response is there a different way to contact github?

**Reply by MathyFey** [reply]
Having the same issue =/
Sent a ticket and it's been a week already =/

**Reply by ppcamp** [reply]
Same here. Did they solve your problem?

**Reply by MathyFey** [reply]
it took a LOOOOOOONG time but yes, they did after contacting support

**Comment by sonersyln** [comment]
They usually respond within 48 hours.
